masterdata/configuration/roles/roles.csv

```diff
@@ -1,4 +1,4 @@
 Uuid,Role name,Description,Inherited roles,Privileges
 ,Lab,"Will give ability to see lab-lite module",,app:lab-lite;Get Orders;Get Patients
-,Nurse,"User have the ability to Register a patient, consultation without prescribing drugs, access Lab, Reports & Partient Documents module.",Provider;Lab,Edit Users;Get Providers;Get Users;Add Encounters;app:registration;Get Concepts;Get Encounters;Get Patients;Get People;Get Visits;Get Visit Types;View Patients;Add Patients;Edit Patients;Edit Patient Identifiers;Get Patient Identifiers;Add Visits;Edit Observations;Add Observations;Edit Encounters;Get Observations;Get Concept Sources;Edit Visits;Get Encounter Roles;Get Visit Attribute Types;app:document-upload;app:patient-documents;app:reports;app:common:closeVisit;Get Care Settings;Send SMS;app:clinical;Get Care Settings;Get Patient Programs;Get Beds;Get Order Frequencies;Get Admission Locations;View Program Attribute Types;Get conditions;Get Diagnoses;app:clinical:observationTab;app:clinical:diagnosisTab;app:clinical:consultationTab;app:clinical:ordersTab;Add Orders;Edit Orders;app:reports;Delete Patient Document;Edit Conditions;Send Email
-,Doctor,"User have the ability to Register a patient, consultation, access Lab, Reports, Admin & Partient Documents module.",Provider;Nurse,app:clinical:treatmentTab;app:admin
\ No newline at end of file
+,Nurse,"User have the ability to Register a patient, consultation without prescribing drugs, access Lab, Reports & Partient Documents module.",Provider;Lab,Edit Users;Get Providers;Get Users;Add Encounters;app:registration;Get Concepts;Get Encounters;Get Patients;Get People;Get Visits;Get Visit Types;View Patients;Add Patients;Edit Patients;Edit Patient Identifiers;Get Patient Identifiers;Add Visits;Edit Observations;Add Observations;Edit Encounters;Get Observations;Get Concept Sources;Edit Visits;Get Encounter Roles;Get Visit Attribute Types;app:document-upload;app:patient-documents;app:reports;app:common:closeVisit;Get Care Settings;Send SMS;app:clinical;Get Care Settings;Get Patient Programs;Get Beds;Get Order Frequencies;Get Admission Locations;View Program Attribute Types;Get conditions;Get Diagnoses;app:clinical:observationTab;app:clinical:diagnosisTab;app:clinical:consultationTab;app:referrals;app:clinical:ordersTab;Add Orders;Edit Orders;app:reports;Delete Patient Document;Edit Conditions;Send Email
+,Doctor,"User have the ability to Register a patient, consultation, access Lab, Reports, Admin & Partient Documents module.",Provider;Nurse,app:clinical:treatmentTab;app:admin;app:referrals
```

Comment:
Here is the Doctor is inserting the Nurse role, so we no need to specifically add the privilege for doctor again, adding it for Nurse is enough.

Reply:
Ok @gokultw will update the code